utilities/tools/Set-ModuleReadMe.ps1

```diff
@@ -346,36 +352,36 @@ function Set-DeploymentExamplesSection {
         [string] $SectionStartIdentifier = '## Deployment examples'
     )
 
+    # Load used function(s)
+    . (Join-Path $PSScriptRoot 'helper' 'ConvertTo-OrderedHashtable.ps1')
+
     # Process content
-    $SectionContent = [System.Collections.ArrayList]@()
+    $SectionContent = [System.Collections.ArrayList]@(
+        'Below is a collection of examples of how this module can be used. These examples are taken from the same files that the CI environment uses to validate this module. The name of each example is based on the name of the file from which it is taken. Also, each example lists all the required parameters first, followed by the rest - each in alphabetical order.',
```

Comment:
I would avoid mentioning the CI environment. examples are anyway in the module .test folder, I'd mention that. Something like:
```suggestion
        'The following module usage examples are retrieved from the content of the files hosted in the module `.test` folder.',
        '   >**Note**: The name of each example is based on the name of the file from which it is taken.',
        '   >**Note**: Each example lists all the required parameters first, followed by the rest - each in alphabetical order.',
```